www-client/firefox/firefox-128.0.2.ebuild

```
@@ -650,7 +651,8 @@ src_prepare() {
 		"${S}"/third_party/libwebrtc/build/toolchain/get_cpu_count.py || die "Failed sedding multiprocessing.cpu_count"
 
 	sed -i -e "s/multiprocessing.cpu_count()/$(makeopts_jobs)/" \
-		"${S}"/third_party/libwebrtc/build/toolchain/get_concurrent_links.py || die "Failed sedding multiprocessing.cpu_count"
+		"${S}"/third_party/libwebrtc/build/toolchain/get_concurrent_links.py \
+		|| die "Failed sedding multiprocessing.cpu_count"
```

You don't need a backlash when the next line starts with `||`,
```suggestion
		"${S}"/third_party/libwebrtc/build/toolchain/get_concurrent_links.py
			|| die "Failed sedding multiprocessing.cpu_count"
```

---

I remembered the syntax wrong, correct form is:
```suggestion
		"${S}"/third_party/libwebrtc/build/toolchain/get_concurrent_links.py ||
            die "Failed sedding multiprocessing.cpu_count"
```
Had you ran **pkgcheck scan** it would've errored due to this :P